src/sciline/visualize.py

```diff
@@ -21,6 +21,14 @@
 from .typing import Graph, Item, Key, get_optional
 
 
+class GraphVisualization(Digraph):  # type: ignore[misc]
```

Comment:
Should this be renamed to `Digraph`? It will make it more obvious that this is essentially the same class?

Reply:
Sure, that's better!